backend/src/services/coordinatorService.js

```
@@ -13,25 +13,46 @@ const {
   });
+  
   const savedDisaster = await disasterRepository.save(newDisaster);
   
+  // Remove coordinator from the response
   const { coordinator: coordinatorData, ...disasterWithoutCoordinator } = savedDisaster;
-
-  return disasterWithoutCoordinator; 
-
+  
+  // Parse coordinates for the response if they exist
+  let responseDisaster = { ...disasterWithoutCoordinator };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'responseDisaster' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)